### ISSUE TITLE
Character Select PRoblem

### ISSUE THREAD

**Issue by HiddenSin** [ValveSoftware/Dota2-Gameplay#2552]
### Description

I am unable to lock in a specific character that I want to play. I am instead given a blank loading screen that informs me to lock in a character but none of the characters pop up and I am forced to Randomize just to get a character.

### Screenshots

![unknown](https://user-images.githubusercontent.com/112574781/187735702-b117f674-1f02-4d3e-bd51-2eefcd9fbd27.png)

**Comment by Mabadman** [comment]
Match ID? Additionally have you tried verifying game files?

**Comment by TheUnholyWrath** [comment]
Try changing "Sort By:" in the bottom left corner. It should let you pick a different grid than the dota plus one

**Comment by Pestamos** [comment]
I would like to second this issue. This screen has ruined my UX as a returning player, because it led to a 30 minute matchmaking cooldown without me even getting into my first match after a very long break from this game. My previous conduct summary from many years ago was all green, so Iwas very surprised by the length of the ban.

After getting to the champ select screen, I have mistakenly clicked on "Sort by: Role Assistant", thinking that it might help me as a player who has not played in a very long time. At that time I did not know that "Role Assistant" required a subscription.

When it got to my turn to pick a hero I got confused that I still couldn't pick. I have previously clicked the "Random" button, but in the meantime an enemy player has locked in the same hero I randomly picked, so I couldn't lock in.

I think that the "Subscribe now" button is a dark pattern, because it does not clearly indicate that the player can switch to the free layout to pick a hero.

In my opinion, there should be another button to go back to the free view below the "Subscribe now" button, or the view should automatically switch when the player is running out of time to pick his hero.

**Comment by Mabadman** [comment]
Fair point. It shouldn't default to dota plus assistant especially if youre not subscribed. Hope this is in-scope of the issue tracker.
